composer.json

```diff
@@ -20,7 +20,9 @@
     "cweagans/composer-patches": "~1.0",
     "derhasi/composer-preserve-paths": "0.1.*",
     "drush/drush": "~8.0",
-    "drupal/drupal": "7.*"
+    "drush/config-extra": "1.*",
```

Comment:
Why `drush/config-extra`? Per https://github.com/drush-ops/config-extra, it's for Drupal 8.x.